warcraft/dapp/algob.config.js

```diff
@@ -0,0 +1,53 @@
+// NOTICE: This config provides sample accounts.
+// DON'T use these accounts in any public environment because everyone can see and use them.
+// The private keys of these accounts are visible to everyone.
+// This means that they can spend the funds and assets.
+
+const { mkAccounts } = require("@algo-builder/algob");
```

Comment:
Do we need this config in dapp? I think here this is a separate UI project. Or is it required by `@algo-builder/web`?

Reply:
right we don't need it